module/x/gravity/migrations/v2/store.go

```diff
@@ -0,0 +1,19 @@
+package v2
+
+import (
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/peggyjv/gravity-bridge/module/v3/x/gravity/types"
+)
+
+func MigrateParamStore(ctx sdktypes.Context, subspace paramstypes.Subspace) error {
+	// Don't want to overwrite values if they were set in an upgrade handler
+	if !subspace.Has(ctx, types.ParamStoreConfirmedOutgoingTxWindow) {
+		subspace.Set(ctx, types.ParamStoreConfirmedOutgoingTxWindow, types.DefaultParams().ConfirmedOutgoingTxWindow)
+	}
+	if !subspace.Has(ctx, types.ParamStoreEventVoteWindow) {
+		subspace.Set(ctx, types.ParamStoreEventVoteWindow, types.DefaultParams().EthereumEventVoteWindow)
```

Comment:
Should the value from `types` here have the same name as the value in `DefaultParams` (i.e. should we change it in the module)?

Reply:
Fixed